x-pack/plugins/security_solution/public/timelines/components/timeline/search_or_filter/index.tsx

```diff
@@ -84,27 +84,29 @@ const StatefulSearchOrFilterComponent = React.memo<Props>(
       getIsDataProviderVisible(state, timelineId)
     );
 
+    // TODO: discuss this
```

Comment:
nit: consider creating an issue about this

Reply:
Looking into it.